packages/venia-ui/lib/components/Header/cartTrigger.js

```diff
@@ -23,23 +34,56 @@ const CartTrigger = props => {
     });
 
     const classes = mergeClasses(defaultClasses, props.classes);
+    const backgroundClass = shoppingBagIsOpen
+        ? classes.background_open
+        : classes.background;
     const buttonAriaLabel = `Toggle mini cart. You have ${itemCount} items in your cart.`;
     const itemCountDisplay = itemCount > 99 ? '99+' : itemCount;
 
     const itemCounter = itemCount ? (
         <span className={classes.counter}>{itemCountDisplay}</span>
     ) : null;
 
-    return (
+    // Because this button behaves differently on desktop and mobile
+    // we render two buttons that differ only in their click handler
+    // and control which one displays via CSS.
+    const cartTrigger = shouldUseShoppingBag ? (
+        <Fragment>
+            <div className={backgroundClass}>
+                <button
+                    aria-label={buttonAriaLabel}
+                    className={classes.trigger}
+                    onClick={handleDesktopClick}
```

Comment:
May want to rename to match CSS class

Reply:
Resolved in https://github.com/magento/pwa-studio/pull/2494/commits/402f24ebfb3c4bb30aaac385881f48845c435332.